tests/languages/arturo/html-string_feature.test

```diff
@@ -0,0 +1,18 @@
+{!html
+    <h1>Header</h1>
+}
+
+----------------------------------------------------
+
+[
+    ["html-string", [
+            ["string", "{!html"],
+            ["embedded", "\r\n    <h1>Header</h1>\r\n"],
```

Comment:
To properly test the HTML highlighting, do this:

1. Declare the optional dependencies in `components.json`. This is necessary for embedded languages to work in general.
2. Create a new folder `tests/languages/arturo!+markup` and move this file into it.

The embedded HTML will then be tokenized in the tests.

Please do this for the other languages as well.

Reply:
All set :)